LectureNotes/Lectures/Fall2018/Lecture4/PPP_notes.tex

```diff
@@ -0,0 +1,186 @@
+
+We would like to answer the following question: how many lights in $A$ are on? To that end, let $N(A)$ denote the number of lights on in $A$ and compute
+\begin{align}
+	\mathbb{E}\left[ N(A) \right] = \mathbb{E} \left[ \sum_{\text{lights in }A} \mathds{1}_{\{\text{lights on}\}} \right]
+	= \sum_{\text{lights in }A} \mathbb{P}\left\{ \text{light is on} \right\}
+		= M \mu(A) \left(\frac{1}{M}\right)
+		= \mu(A).
+\end{align}
+Thus $\mu$ gives the expected density for the set of lights that are on in $A$. By construction, we know $N(A) \sim \operatorname{Binom}\left(M\mu(A),M^{-1}\right)$, and hence $N(A) \sim \operatorname{Pois}(\mu(A))$. To see this, put $L = M\mu(A)$ and observe,
+\begin{align}
+	\mathbb{P}\left\{N(A) = n \right\} &= \binom{L}{n}\left(\frac{1}{M}\right)^n\left(1 - \frac{1}{M}\right)^{L-n}\\
+	&= \frac{L(L-1)\cdots (L-n+1)}{n! M^n} \left( 1 - \frac{1}{M} \right)^{L-n}\\
+	&\simeq \frac{1}{n!}\left( \frac{L}{M} \right)^n \exp\left( -\frac{L}{M} \right) + \mathcal{O}\left( \frac{1}{M} \right)\\
+	&\simeq \frac{\mu(A)^n}{n!} e^{-\mu(A)}
+\end{align}  
+This motivates the following definition.
+\begin{definition}
+	Let $\mu$ be a $\sigma$-finite measure on some space $X$. A \emph{Poisson Point Process} (PPP) on $X$ with \emph{mean measure} (or, \emph{intensity}) $\mu$ is a random point measure $N$ such that:
+	\begin{enumerate}[(a)]
+		\item For any Borel set $A\subset X$, we have $N(A) \in \mathbb{Z}_{\geq 0}$ and $N(A) \sim \operatorname{Pois}(\mu(A))$, i.e.
+		\begin{align}\label{eq:ppp}
+			\mathbb{P}\left\{ N(A) = n \right\} = \frac{\mu(A)^n}{n!} e^{-\mu(A)}.
+		\end{align}
+		\item If $A$ and $B$ are disjoint Borel subsets of $X$, then $N(A)$ and $N(B)$ are independent random variables.
+	\end{enumerate}
+\end{definition}
+Recall a point measure is just a measure whose mass is atomic. That is, if $\{x_i\}\subset X$ then a point measure is of the form
+\begin{align}
+	\mu = \sum_i a_i \delta_{x_i}
+\end{align}
+where $\delta_x$ is the unit point mass at $x$.
+\section{PPP Properties}
+It is sometimes useful to think of a PPP as a random collection of points. With this in mind, we list some important properties of $N\sim \operatorname{PPP}(\mu)$ on some space $X$:
+\begin{itemize}
+	\item \emph{Enumeration:} It is always possible to enumerate the points of $N$, i.e. there is a random collection of points $\{x_i\}\subset X$ such that
+	\begin{align}
+		N = \sum_i \delta_{x_i}.
+	\end{align}
+	\item \emph{Mean measure:} If $f\colon X \to \mathbb{R}$ then 
+	\begin{align}\label{eq:mm}
+		\mathbb{E}\left[ \int\! f(x) \, dN(x) \right] = \int f(x) d\mu(x).
+	\end{align}
+	\underline{Note:} This is a more general property of point processes, as any point process has a mean measure. To see \eqref{eq:mm} holds without needing $N$ to be a \emph{Poisson} point process, let $f$ be a simple function, i.e.
+	\begin{align}
+		f(x) = \sum_{i=1}^n f_i \mathds{1}_{A_i}(x), \qquad\text{where}\qquad X = \bigsqcup_i A_i.
+	\end{align}

```

Comment:
Perhaps clarify that $\bigsqcup$ means a *disjoint* union, i.e., that the $A_i$ are pairwise disjoint.